src/download.js

```diff
@@ -1,13 +1,19 @@
-var zip = require("./zip");
+var zip = require('./zip');
+var saveAs = require("file-saver").saveAs;
 
-module.exports = async function (gj, options) {
-  var content = await zip(gj, options);
-  var link = document.createElement("a");
-  link.href = "data:application/zip;base64," + content;
-  link.rel = "noopener";
-  if ((options && options.filename) || options.folder) {
-    link.download = (options.filename || options.folder) + ".zip";
+module.exports = function (
+    gj, 
+    options, 
+    zipOptions = {
```

Comment:
I'm really starting to like the idea of just removing the `download` function all together, and just doing `shp` stuff in this library. Might do that in a future release...

Reply:
That is an option, however, I think users would appreciate a snippet of how to download the files with some suggested methods in the README since that is likely the most common use case.